plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/model/Library.java

```diff
@@ -115,7 +115,7 @@ void setSiteUri(URI siteUri) {
   /**
    * @param libraryFiles artifacts associated with this library, cannot be <code>null</code>
    */
-  synchronized void setLibraryFiles(List<LibraryFile> libraryFiles) {
+  public synchronized void setLibraryFiles(Collection<LibraryFile> libraryFiles) {
```

Comment:
This came because I was using a set to remove the duplicates. Should we sort the set into a list? Perhaps alphabetically by jar name? That would certainly make the classpath a little easier to scan in the package explorer.

Reply:
reverted to list, sorted roughly by maven coordinates